Test/SqlDom/TSqlParserTest.cs

```diff
@@ -426,6 +426,37 @@ public void OpenRowsetBulkWithThreeFiles()
             ));
         }
 
+        [TestMethod]
+        [Priority(0)]
+        [Timeout(GlobalConstants.DefaultTestTimeout)]
+        [SqlStudioTestCategory(Category.UnitTest)]
```

Comment:
You can add a test to [Only120SyntaxTests](https://github.com/microsoft/SqlScriptDOM/blob/main/Test/SqlDom/Only120SyntaxTests.cs) (the version where the new grammar is introduced) instead of writing a new one. It runs parser and script generator. You just have to add a sql file to TestScripts and Baselines120 folder.

Reply:
I add a test  there too but keep this one as well. the problem was with the startcolumn so I want to specifically verify it here

Reply:
I verified those tests already verifying the atomic statement https://github.com/microsoft/SqlScriptDOM/blob/1c3a5cfe8289fda5c45f7b4f75fad8227508fc7f/Test/SqlDom/Baselines120/CreateProcedureStatementTests120.sql#L41